pykrige/tests/test_api.py

```diff
@@ -46,3 +50,43 @@ def test_krige():
                 assert estimator.best_score_ > -3.0
         if hasattr(estimator, 'cv_results_'):
             assert estimator.cv_results_['mean_train_score'] > 0
+
+
+@pytest.mark.skipif(not SKLEARN_INSTALLED,
+                    reason="scikit-learn not installed")
+def test_gridsearch_cv_variogram_parameters():
+    # First dictionary with a dictionary in the place of the variogram_parameters
+    param_dict3d_1 = {"method": ["ordinary3d"], "variogram_model": ["linear"],
+                 "variogram_parameters": [{'slope': 1.0, 'nugget': 1.0},
+                                          {'slope': 2.0, 'nugget': 1.0}]
+               }
+    # First dictionary with a list of lists in the place of the variogram_parameters
+    param_dict3d_2 = {"method": ["ordinary3d"], "variogram_model": ["linear"],
+                 "variogram_parameters": [[1.0,1.0],
+                                          [2.0,1.0]]
+               }
+
+    estimator1 = GridSearchCV(Krige(), param_dict3d_1, verbose=True)
+    estimator2 = GridSearchCV(Krige(), param_dict3d_2, verbose=True)
+
+    # dummy data
+    np.random.seed(42)
+    X3 = np.random.randint(0, 400, size=(100, 3)).astype(float)
```

Comment:
I know this is used in the CV example, but in retrospective, I do find kriging on integers quite strange. Could we not make this,
```py
X3 = 400. * (1 + rng.rand(100, 3))
```
more simply
?